src/amruta/Assignment_15/BankusingSwitch.java

```diff
@@ -0,0 +1,210 @@
+
+/*
+
+Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+      a) Person can debit or credit amount from respective bank.
+          Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+      b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+      c) A Method to show total credit, debit operations 
+             output : Total credit operation - 7 times
+                          Total debit operation - 5 times
+
+     d) A Method to show individual bank credit operations.
+            output : ICICI bank Credit Operation - 2
+                     HDFC bank Credit Operation - 2
+	                 SBI bank Credit Operation - 1
+                     BOB bank Credit Operation - 4
+
+     e) A Method to show individual bank debit operations.
+            output : ICICI bank Debit Operation - 2
+                     HDFC bank Debit Operation - 2
+	                 SBI
+
+*/
+
+package amruta.Assignment_15;
+
+import java.util.Scanner;
+
+class BankusingSwitch{
+double iciciBal = 10000;
+double hdfcBal = 30000;
+double sbiBal = 50000;
+double bobBal = 40000;
+
+int icicidebitcount, hdfcdebitcount, sbidebitcount, bobcreditcount ;
+int icicicreditcount, hdfccreditcount, sbicreditcount, bobdebitcount;
+
+static int totaldebitcount;
+static int totalcreditcount;
+
+void operation(String bankname, String operation, double amount){
+	
+	switch(operation.toUpperCase()){
+	
+		//case "debit" :
+		case "DEBIT" :
+			switch(bankname.toUpperCase()){
+			//	case "icici" :
+				case "ICICI" :
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)